woocommerce/woocommerce
# Launch Your Store: Add useLaunchYourStore helper method

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Relates to https://github.com/woocommerce/woocommerce/issues/45111

While adding the status pill from the issue above, I needed first to do some reorganising. This PR adds the non-functional status pill and adds a `useLaunchYourStore` method. Development is in progress but behind a feature flag, so I wanted to get this progress merged so far to keep the PR size small.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Spin up a new JN site and turn on the feature flag. Tools > WCA Test Helper > Features > launch-your-store
2. Visit any Woo page such as `/wp-admin/admin.php?page=wc-settings`. See the status pill (non functional at this time) appear.
<img width="616" alt="image" src="https://github.com/woocommerce/woocommerce/assets/1922453/b19a663e-44a6-4480-b37e-2fe59bd16ff6">

3. Confirm Site visibility section loads. You should be able to manipulate and save settings within.
<img width="733" alt="image" src="https://github.com/woocommerce/woocommerce/assets/1922453/b8e492b3-e654-4ba8-b78f-9ada85a40e61">




### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

Updates unreleased file structure for LYS

</details>
